meson.build

```diff
@@ -2,9 +2,11 @@ project('appstream-glib', 'c',
   version : '0.7.19',
   license : 'LGPL-2.1+',
   default_options : ['warning_level=1', 'c_std=c99'],
-  meson_version : '>=0.37.0'
+  meson_version : '>=0.46.0'
```

Comment:
do we really need this new version?

Reply:
With the older meson version requirement, this warnings are show:

```
WARNING: Project specifies a minimum meson_version '>=0.37.0' but uses features which were added in newer versions:
 * 0.46.0: {'compiler.get_supported_link_arguments_method', 'compiler.has_multi_link_argument', 'compiler.has_link_argument'}
 * 0.50.0: {'install arg in configure_file'}
```

meson 0.46.0 was released in Apr 23, 2018

Reply:
> meson 0.46.0 was released in Apr 23, 2018

Fair point.